domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/Link.kt

```diff
@@ -0,0 +1,5 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+data class Link(
+    val href: String?
```

Comment:
@VanyaHuamanBottleRocket nit: Add trailing comma for less noisy future diffs :)

```suggestion
    val href: String?,
```

Reply:
Updating